salt/modules/boto_elb.py

```diff
@@ -686,3 +694,223 @@ def get_instance_health(name, region=None, key=None, keyid=None, profile=None, i
     except boto.exception.BotoServerError as error:
         log.debug(error)
         return []
+
+
+def get_listener_policy(name, listener, region=None, key=None, keyid=None, profile=None):
+    '''
+    gets the policy details of a specific listener port
+
+    name
+        name of the elb
+
+    listener
+        the listener port to retrieve details on
+    '''
+    conn = _get_conn(region=region, key=key, keyid=keyid, profile=profile)
+    lb = conn.get_all_load_balancers(load_balancer_names=[name])
+    lb = lb[0]
+    for _listener in lb.listeners:
+        if _listener.load_balancer_port == listener:
+            log.debug('found the listener')
+            if _listener.policy_names:
+                ret = _get_listener_policy_info(lb, _listener.policy_names[0])
+                return ret
+            else:
+                return 'The specified listener does not have any policies.'
+    return 'The specified listener was not found.'
+
+
+def _get_listener_policy_info(lb_object, policy_name):
+    '''
+    gets the details of a policy name
+    '''
+    lb_cookie_policies = lb_object.policies.lb_cookie_stickiness_policies
+    app_cookie_policies = lb_object.policies.app_cookie_stickiness_policies
+
+    for p in lb_cookie_policies:
+        if p.policy_name == policy_name:
+            return {'policy': 'LBCookieStickinessPolicy', 'name': p.policy_name, 'cookie': p.cookie_expiration_period}
+    for p in app_cookie_policies:
+        if p.policy_name == policy_name:
+            return {'policy': 'AppCookieStickiness', 'name': p.policy_name, 'cookie': p.cookie_name}
+    return None
+
+
+def set_listener_policy(name, listener, policy, cookie, disable_policy=False, region=None, key=None, keyid=None, profile=None):
+    '''
+    sets a sticky session policy to a listener port
+
```

Comment:
A versionadded tag should be added here, too, as well as to all of your new functions below. (private functions don't need them, but public ones do) for our docs.

All of these new functions should also have CLI examples, as mentioned above.